applications/tari_merge_mining_proxy/src/common/json_rpc.rs

```diff
@@ -23,6 +23,18 @@
 use json::json;
 use serde_json as json;
 
+/// Default accept response for submit block that goes back to XMRig
+/// Refer to XMRig. Do not change existing json values unless it is changed in XMRig.
+pub fn default_block_accept_response(req_id: Option<i64>) -> json::Value {
+    json!({
+       "id": req_id.unwrap_or(-1),
+       "jsonrpc": "2.0",
+       "result": "{}",
+       "status": "OK",
```

Comment:
Strange that the status is not in the result object, it should be since that is the way every other call works i.e. the "outputs" are in the result key as per the JSON RPC spec. `"untrusted"`  

https://www.getmonero.org/resources/developer-guides/daemon-rpc.html#submit_block 

Any links on hand that you could provide to detail "xmrig flavour" RPC?
Might want to include a comment above `"result": "{}",` saying that this MUST be a string due to a bug in xmrig, preferably with an github issue (quick search I couldn't find anything)

  
Using POSTman, monerod includes these fields in the `result` object.

![image](https://user-images.githubusercontent.com/1057902/109170465-78739e00-7789-11eb-92dc-27c9a74265a7.png)

Reply:
Embedding it will work as well.
```
       "result": {
           "status": "OK",
           "untrusted": false
       }
```

It just cannot evaluate to null (caused by this: `json!({})`), which is fixed again in this PR.

Reply:
Will update

Reply:
Done

Reply:
Actually no, that only works for solo mining, not self-select. Reverting.

Reply:
Done, as-is it works for both cases.